docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "2.3"
+services:
+  detectron2:
+    build: .
+    runtime: nvidia  # TODO: Exchange with "gpu: all" in the future (see https://github.com/facebookresearch/detectron2/pull/197/commits/00545e1f376918db4a8ce264d427a07c1e896c5a).
+    shm_size: "8gb"
+    ulimits:
+      memlock: -1
+      stack: 67108864
+    volumes:
+      - /tmp:/tmp
```

Comment:
how about "/tmp/.X11-unix"?

Reply:
I'm glad you like it. You are absolutely right about the model caching. The best solution depends on, whether you would like to enable caching of downloaded models or not.
If yes, then there are two options:
1. Wait for #383 to be merged. Then the current version would enable model caching.
2. Add
```
  environment:
    FVCORE_CACHE=/tmp
```
to the docker-compose file, to support model caching without #383 being merged before.

If not, then we should make the change you suggest.

Reply:
The models are quite large (depend on how many you'll use) so I'm not sure using `/tmp` is the best option to cache models. Is it feasible to mount `$HOME/.torch/fvcore_cache`?